### ISSUE TITLE
[US - DEV] As MILES HARDIN, I want to edit or remove a comment because I made a mistake and want to fix it

### ISSUE THREAD

**Issue by JulieMass** [dev-launchers/dev-launchers-platform#858]

**Comment by JulieMass** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/IdeaSpace-Team-6350185d9843157427cc900b/issues/dev-launchers/dev-launchers-platform/858?planning-poker) with Zenhub @alicefate @dbradham @jordanmcousins @JulieMass @katehirschman @MariliaDP @Nouri-Anouar @rashikashaw @RETprojects

**Comment by JulieMass** [comment]
@dbradham , @jordanmcousins @katehirschman @MariliaDP @alicefate @RETprojects Hey everyone! Could you take a quick look at your estimates for this ticket? We've made some changes and split it into two separate tickets - one for editing comments and another for removing them. Let me know if you need to update your estimates. Thanks!

Also, another question I have for all: shouldn't we have an action to cancel while the input becomes editable? We didn't cover this in our designs, so this is what I wrote in the acceptance criteria: "If she clicks outside the comment, any changes she made to the comment should be discarded, and the comment should remain unchanged." but is this the best solution or should we have a cancel button? 

Let me know your thoughts please,

**Comment by MariliaDP** [comment]
@JulieMass, I don't think we need a cancel button. If she realizes she doesn't need any changes to her comment, she can still click the "-> " save button to confirm her comment or click outside.

**Comment by JulieMass** [comment]
> @JulieMass, I don't think we need a cancel button. If she realizes she doesn't need any changes to her comment, she can still click the "-> " save button to confirm her comment or click outside.

Thank you Marilia for your feedback. I'll leave the AC as it is now then =)
